### DISCUSSION TITLE
Allow data editing in explorations

### DISCUSSION THREAD

**Discussion by kgodey** [mathesar-foundation/mathesar#2280] [Feature Ideas]
Currently, only data in tables can be edited.

We should support editing data in explorations.

---
*If you'd like us to build this feature, please comment below with more details about how you plan to use this functionality.*

**Comment by jeromeperez** [comment]
Hi
This feature should be interesting only if modification are repercuted into database and only for administrator or editor.

**Reply by seancolsen** [reply]
@jeromeperez Thanks for the clarifying feedback.

> modification are repercuted into database

Yes, that's how we're envisioning this feature to work.

> only for administrator or editor

Also yes. Only users with the appropriate permissions would be able to edit.
